db/migrate/20210113152529_add_cavc_remand_cavc_appeal_id.rb

```diff
@@ -0,0 +1,9 @@
+class AddCavcRemandCavcAppealId < Caseflow::Migration
+  def up
+    add_column  :cavc_remands, :remand_appeal_id, :bigint, comment: "Court remand appeal created by this entry"
```

Comment:
Suggestion to reduce ambiguity:
```suggestion
    add_column  :cavc_remands, :remand_appeal_id, :bigint, comment: "Appeal created by this CAVC Remand"
```